src/helpers/util.js

```diff
@@ -45,6 +45,21 @@ const uniqueFiles = (files, newArray) =>
     newArray.filter(file =>
       !files.map(file => file.path).includes(file.path)))
 
+const getUniqueExtensions = files => {
+    const normalizeExtension = name => {
```

Comment:
You could probably declare `normalizeExtension` outside of this function (:

Reply:
Sorry I didn't make it :(

Reply:
Don't worry about it 😄I was just nitpicking. Update not necessary. Thanks a lot for this bug fix! We are going to release today.

Reply:
Go for it! 👍 that would be massively appreciated. Also thanks so much for writing that test! We really should have been better about doing so

Reply:
No shame! TDD is great but it takes time that I usually do not have to commit 😂